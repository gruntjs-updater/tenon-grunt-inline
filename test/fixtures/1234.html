<!DOCTYPE html>
<html>
  <head>
    <title></title>
    <script src="1234.js"></script>
    <link href="1234.css" rel="stylesheet" type="text/css" />
  </head>

  <body>
    <img src="1234.png" />
  </body>
</html>
