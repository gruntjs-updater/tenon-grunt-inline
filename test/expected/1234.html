<!DOCTYPE html>
<html>
  <head>
    <title></title>
    <script>console.log("Hallo!");
</script>
    <style>body > img {
  border: 3px solid red;
}
</style>
  </head>

  <body>
    <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAdIAAAAiCAYAAADrhVibAAAABHNCSVQICAgIfAhkiAAADJBJREFUeF7tnXtsHMUZwGdmZ+/98PlNEmLHeTUB8nADCQ3QEDdQ2groCyRU1D9K6VNt1QpoS2mFVCgSjwpVFLVSaStatUVqgYAqIRAQEiCkJDZOoMqDxI6dNH6dffa9d3em3158yfk8e44dO/jsbyXLvm9nv/nm9633u2929xtKCDkJPzO/MUqW3surQ38zYq1HiHFeHVJC/Iupbp2QZtok8rx0aZQsv49Xh/9hDO89Qc2FTUTvbZcZw8fYpT8D+bPZ4V3vkPR59YEHIwEkgASQwJwlwPMj8zZrnuYvsaBfJ5QMCeuD31lDvQ1cv3aL8O14TETTEK74JdzdcpMI7HxIDMh149t3J5m48kGt1jggM7yaaiFd0PbHrWhfraZtvJWFA4uozr+qVwRTUg5vN0faDpCMfzP3Xn4TDboEITIpxf7HzOipOIFP6i24jfvWbqS+4ELCD91n9H3YTyy7ZSk9nss098qV0nX4WTGSzLU+u7GFTF/sFqztKDFkRsruNpLJ7Y0L0d9LzNowhVArzzNaj+0TPyEBJIAEkMDcIZALpLSKaeu/QIOdjxrR7gFi+bZw31W3slDfn0Vs8BYWqgsJ1hkjovJq6s28JVPpCNM2Kdr3/InECCfU3GEm3j1MjLpv6ZFVzcLT87qVfOtXon9cRuqitPEGGog+aQy2HyMGC1Cmp0vHrJGXzeSuN2hq7S949Rk3TKBHb2B64ybh7dpO4hBIx2SwFVuYj7WaqcHMWDmtYdqSBZJ3/lXGzi/lnTsnC44ECSABJIAExhPIBVK+mPFIJdX4nXrFElugESqTRPCUEJ2dmrHiUuo+3krTi5dIV8czckRr0JTtXTDlSiCr6+kmpq0mDRmdVkkgo4Ns0xYUb4aU0S5irLuNh0SrSPe3iUxP/PSxxU1Lfp5Az8iLRvz5F0l8nA4fo0vWEHf3r2V0TArso/Rjd2oR+YIZP9pzOuMddywKkAASQAJIAAkAgTNTu7LLMvbAlK09hVu4ZXfKpPd66g8kmKjsF+b7gxBYGghXtofAZEfMvAr7N7WjqNMGDf73e2MotpTptZcy9+p7tKrax41oO0yzOh2ilE9Rj2sVc9cMC7HrVEHwhuy26Zs8UnvATL/5hkw6zjErDUEhEkACSAAJzDcCzB6w2SXMoRqNNzYS3f5MvZTWLKO6vdM4IrIDVRpf8WkaSL0J07oQtEq1dwRoHyeI9PhIrs/cBn8F6ghPHxFGx3Yr/uEpaoYjkA+Pbp41mnvdLSzoOyPJ7yn6PRU9EOHrWpg/s0skE/n7ppA7X/w1Hm44YZm7t8u4WfSlwqF3FCMBJIAEkMA8JpDLSGW/sFr/QmPNX9crroNpXTuLHHzJjA/YT9emhDx+lGevhHudrfAQUS62OLU/WWJaVkrS+4pMNN2uh7fCtHHsn8bI3g9otvbLPLjsYgjgEAytY5ax7/3Rh32gG30x0xuuEN7jz42/tznGZ4zSyeqh1UxrXCD4sbbRMYFCrZHpq9ZTryvLZctVxGf3kXrFiO94XsYxMx1DHD8gASSABJDAKAF74vXCvP4ynchDjG36Oas5cb/Z1zXi/ITvdHaJupAAEkACSAAJqAicuUeq2jnrZH7Gmu/WKquDRMvusVK9JV6TmXW2o0FIAAkgASQwJwmUZ0Y6J12Bg0ICSAAJIIFyJHD2wZ9ytB5tRgJIAAkgASTwERPAQPoROwC7RwJIAAkggfImMHcCKUxSV27VfBdBAYgzLoH3Wq98hNfVe6Hs4VzdYGR139ArPtlCfWMG6SQv4KCt5K7rH9BqwlCNyhb7WnTfZ+9ilS4nVjPEM/IVPbT1BurX6hnfcK9e9aknXfXXNBOPkxlO8ryeiU5qvoq7rvthiXE6dTBD8lllj+r/aIbGjWqRwFwhMNE1p3zGCe/sRDYzX31lwXuq5WP9lC21X+NZ3ihdB3fLMbU0nORT7ugCHChPCfPdB4yB/f+Vp+sdX4A+sYsiAvP0/wjPAyRwPgTK66ldh5FqyzRdWRT/KMkSKLZUf5seumQ19bjjlrXvUSt6apgIVq/xq79PK+Fd2Uykjuo6l+T9R8yBxAbu2XAjDbqhLoXZIbKtT4vY4nt4VewRI3oserZcYN139MjSt43UW/vUK8OoFgHohgL7Tv32JIkMfk4PbIVFAnb+1OqLZtVVFYsRVGzV/J69Zqo3MfY1ICd58fHKz3bmqVh84JS90o6KZ5LKhjt4eEUTyVWJNDuF0faUNRSljCj1wCtLkRu4f8N1LEBiQgyliaBQ41lpy6hQyROOmaweW52EShtpeOK78N1gpf4S/uoLado5nT9/AA4pIkv5XWWPEwuVnRdicQkne1COBJBAQYnAcoZhHbEMZVF8H0xZeih1tRmZV/9IYot+wCtPF9GHSsKwsQjV3HuM9K79JCZhelMLM7bxZho68bAxcLiHmou+yyPrIbAd7CPWwosI70hTUVkltWgPtaprCe+FOsEqbk6LAPT+VgzaRYiL+2WjhfQNCEAdu6F6VFFhfVUfOVmQseXrpefIw3JgTMEIJ7mjIsUO1eIDe0hKyXOHTPb9yxjp6iWWgEhae4desfYTwvv629Beoaf3IMuuaSGBQ7/M9ncmqVhzv15TamrEiWf/s3RkMnryo7QOW9k3DsOXrNHNSX9Jf4XG+1F5/mwT/tegSpbdlZPfi+1ReCMncrJzxheXcDII5UgACeQIzImMtKQv00J2w1ShhEgzfJwY+SL69jGwrovVeYhkc9Wa7GwLVnwJDAurHYrtEwGVmN4hqbVbqD/ZTtL+FVT313Ft47XC/+ZvZDRCBfkAVsRR9e20CIAOQcYOpMX95pWk91uZ9/afreyk0l0oC2zWvBWHrcy+gkzZ3u8kV+kbVwUxL3BYfIA48PR+nHuaL6deSOxzAUOHrD9341Whx3UR5QGo23zKzkKh4tXJQzJbozJuVObE07NocnqcunDSX8pfduAv9qPT+aO/IOP2N67i9sqTx8lIkDvZOeOLS5SwCXchASQwHwIpXK0gJuY2uGbD1/qzbpdZKa3CSFKwr/DkgFrAWXozC9aMSNHVTY261dTt6RFmvMT0q7KoPyjNXYCL+x3tjEL25vIQahRNOxbacuZvyLSbriG+409Alls4Bie5SkkujI3dJFzxcxd42JHfl8OWZ6PgqS3V9OZrpP+9B8x+u0hG+It68ArXKGkHPbmORzuQisXZx9mlWFRBb2aeifSohq2STcVf4/zocP7k+xvXXmXIBDInO2d0cYkJbMLdSGC+Eyg1o1ZebODKO64o/iRHIHqEFQ8xrb4GMnW4F1hzBfGmIZtNn5RWvIbxJliovBOy1AUt1G+CXDmvC31Oqag/HOfZrHuvf5DXVkMwnch093rNU99rWR1F9Y2d5Cp9YkiItJeySAXEd+gxvJy4jG5pTDZTorBiDktIEYcaynCXlNatoZ5S6wyYJ6WZCDMtbD9NzSmtaqKuwhPRhGDsqqNaXubEU0ygRzVmlcxJ/2T/OZzOH6P4W4HKiHOQlbJzRheXOAfbsAkSmM8E5s7Urqoo/pGz98HOxcmwKqnV/hwb3vAjvbIJLn5WlzBaX5YJAYt+98G9vMBBmY0fFUamQtMSx6Tj9bHkIgAlDPEsJNw6JrIjJTLd3OEQfBq2UX/f38XwmEd1neQOfcKdVWv/v9nI+h+7qlfBmWB2WEbbS3JYTBzHx2g04R71oSi3Nt3NqpIQTLOD0ioVjEWPZbbv4ol1d/GqZTAFbE+7F7YffEkkEt/TK7dtkf7BZ4zh/+wVaeWiCq9ayVJ6HIY9TjxVfxUrcjx/oOFkg3KxbvtzSTtncnGJ9nO/3aCyG2VIYK4TsDOf8itaPxe9Aku4LfsJr/Y8bcQOdKofYsoPm6/mri2fl6HdD1n98YJnXZ3kcxEXjgkJIAEkMFsIYCCdLZ5AO5AAEkACSKAsCUzHjFNZDhyNRgJIAAkgASQwHQQwkE4HRdSBBJAAEkAC85YABtJ563ocOBJAAkgACUwHAQyk00FRpQNen1n0bVfkxif0+uUL5kHhCxUDlCEBJIAE5gEBDKQz5OTwZ3hgJRRR3fmUHFp0h1ZR45+WNyBmyFpUiwSQABJAAlMlgIF0quRKHOe6THOvWy5c7z4lYoN7zfSe12jikttZyIe0S1DDXUgACSCB8iSAr7+Up9/QaiSABJAAEpglBDBHmiWOQDOQABJAAkigPAlgIC1Pv6HVSAAJIAEkMEsIYCCdJY5AM5AAEkACSKA8CWAgLU+/odVIAAkgASQwSwhgIJ0ljkAzkAASQAJIoDwJ/B/1waDvBJ+1xwAAAABJRU5ErkJggg==">
  </body>
</html>
